app/routes/task_routes.py

```diff
@@ -1 +1,63 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response
+from app.models.task import Task
+from ..db import db
+from .route_utilities import create_model, get_models_with_filters, validate_model
+from datetime import datetime
+from .slack_routes import send_slack_notification
+
+bp = Blueprint("task_list_bp", __name__, url_prefix="/tasks")
+
+@bp.post("")
+def create_task():
+    request_body = request.get_json()
+    return create_model(Task, request_body)
+
+@bp.get("")
+def get_all_tasks():
+    return get_models_with_filters(Task, request.args)
+
+@bp.get("/<task_id>")
+def get_one_task(task_id):
+    task = validate_model(Task, task_id)
+    result = {}
+    result["task"] = task.to_dict()
+    return result
+
+@bp.put("/<task_id>")
+def update_task(task_id):
+    task = validate_model(Task, task_id)
+    request_body = request.get_json()
+
+    task.title = request_body["title"]
+    task.description = request_body["description"]
+    
+    db.session.commit()
+
+    return Response(status=204, mimetype="application/json")
```

Comment:
This is another opportunity to D.R.Y. up our code! Notice how in this route and the `PUT` route for `goal_routes.py` we follow the pattern of:
```py
object.ATTRIBUTE = request_body["ATTRIBUTE"]
```
We use `hasattr` and [setattr](https://docs.python.org/3/library/functions.html#setattr) to make a helper function to update our `Task` and `Goal` model. It would look like this: 
```py
def update_model(obj, data):
    for attr, value in data.items():
        if hasattr(obj, attr):
            setattr(obj, attr, value)
    
    db.session.commit()

    return Response(status=204, mimetype="application/json")
```

This refactor not only makes our code D.R.Y but shows that we recognize logic that has higher level usability while handling cases of keys not being found!